Source/Csla.Generators/readme.md

```diff
@@ -0,0 +1,97 @@
+# Using the Source Generator with AddressPOCO in CSLA.NET
+
+This guide provides a step-by-step approach to using the source generator in the CSLA.NET project, specifically with the `AddressPOCO.cs` class as an example. The source generator is a powerful tool that can automatically generate additional source code for your projects, enhancing productivity and maintainability.
+
+## Prerequisites
+
+- Visual Studio 2019 or later with the .NET Core development workload installed.
+- The CSLA.NET NuGet package added to your project.
+- Ths CSLA.NET Source Generator NuGet package added to your project.
```

Comment:
Is this being built now?

There's a PR I hope to close today that changes the nuget package creation to be based on the csproj files, so this will need to work within that model.

Reply:
I created a base PR of the nuget packages, After this merge I can add to the project do generate this project as nuget package also